src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/OidcUserInfoTokenHandlerFactory.php

```diff
@@ -60,10 +60,13 @@ public function addConfiguration(NodeBuilder $node): void
                     ->scalarNode('base_uri')
                         ->info('Base URI of the userinfo endpoint on the OIDC server.')
                         ->isRequired()
+                        ->cannotBeEmpty()
                     ->end()
                     ->scalarNode('claim')
                         ->info('Claim which contains the user identifier (e.g. sub, email, etc.).')
                         ->defaultValue('sub')
+                        ->isRequired()
```

Comment:
does this one needs to be required if it has a default value?

Reply:
Nop, removed in a squashed commit. Tests revealed the bug too.
Thanks for noticing :wink: